ecosystem/README.md

```diff
@@ -1,14 +1,5 @@
 # Stellar Ecosystem Proposals (SEPs)
 
-## SEP Tracks
-* **Informational** — A SEP on the `Informational` track is one that is open to adoption by the
-  ecosystem, but has not been formally standardized by SDF, and is not endorsed by SDF for
-  adoption. Typically a SEP can start as `Informational` to gain traction within the ecosystem
-  before moving to the `Standards` track.
-* **Standard** — A SEP on the `Standards` track is one that aims for formal standardization and
-  endorsement by SDF for adoption. Typically a SEP Standard has a higher bar towards acceptance,
-  and it requires approval by 2 SDF members of the SEP Team.
-
```

Comment:
The word `Standard` is mentioned in the SEP Status Terms section. I can't comment on the lines directly, but its line 12, line 15, line 19.

Reply:
updated